learn-ocaml.el

```diff
@@ -156,8 +155,89 @@
      :sentinel (apply-partially #'learn-ocaml-error-handler
 				buffer
 				(lambda (s) (funcall-interactively callback (replace-regexp-in-string "\n\\'" "" s))))))) 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+
+;;;;;;;;;;;;;;;;;;;wrappers ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defun learn-ocaml-show-metadata ()
+  (interactive)
+  (learn-ocaml-give-token
+   (lambda (token)
+     (learn-ocaml-give-server
+      (lambda (server)
+  (message-box " Current token: %s \n Current server: %s" token server))))))
 
 
+(defun learn-ocaml-create-token-wrapper (nickname secret)
+  (interactive "sWhat nickname you want to use for the token ? \nsWhat secret do you want to associate to this token? " )
+  (learn-ocaml-create-token nickname secret
+			    (lambda (token) (learn-ocaml-use-metadata token nil
```

Comment:
style detail: feel free to add a newline after `(learn-ocaml-create-token` and any other similar occurrence, to expend less indentation :-)